### DISCUSSION TITLE
how to restore a deleted team

### DISCUSSION THREAD

**Discussion by SarikAnwar01** [vercel/community#6570] [Account Issue] [answered]
### Question

how to restore a deleted team

**Comment by amyegan** [comment, marked as answer]
Hey @SarikAnwar01. A deleted team cannot be restored, but you can create a new one to deploy the same code

**Reply by SarikAnwar01** [reply]
i mean I have to downgrade from pro to hobby so I have to merge it with my personal hooby  team , but I thought if I could make a hobby team and then I can downgrade the current team and merge it to the new  hobby team so it will not interfere with personal and other hobby account but I am unable to create new hobby account @amyegan  can u help me 
with this
